rocks-lib/src/lua_package/mod.rs

```diff
@@ -130,16 +130,11 @@ impl<'de> Deserialize<'de> for LuaPackageReq {
 
 /// A luarocks package name, which is always lowercase
 #[derive(Debug, PartialEq, Eq, PartialOrd, Ord, Clone, Hash)]
-pub struct PackageName {
-    name: String,
-}
+pub struct PackageName(String);
```

Comment:
Wrapping these up as [newtypes](https://doc.rust-lang.org/rust-by-example/generics/new_types.html), since this is what these are at the end of the day.